test/class.js

```diff
@@ -15,6 +20,22 @@ describe('Class helpers', () => {
     expect(el.className).to.contain('my-class')
   })
 
+  it('should add a class properly when using a fallback path', () => {
+    var el = document.getElementById('item-1')
+    removeProperty('classList', el)
```

Comment:
I can, but this change is only on the instance and not on the prototype itself and u make fresh nodes from fixtures in beforeEach hook anyway.